app/src/main/java/ar/scacchipa/twittercloneapp/domain/usecase/AuthorizationUseCase.kt

```diff
@@ -1,21 +1,31 @@
 package ar.scacchipa.twittercloneapp.domain.usecase
 
 import ar.scacchipa.twittercloneapp.data.repository.IAuthorizationRepository
+import ar.scacchipa.twittercloneapp.data.repository.ICredentialRepository
+import ar.scacchipa.twittercloneapp.domain.model.Credential
 import ar.scacchipa.twittercloneapp.domain.model.ResponseDomain
 import ar.scacchipa.twittercloneapp.utils.Constants
 
 open class AuthorizationUseCase(
-    private val repository: IAuthorizationRepository
+    private val authRepository: IAuthorizationRepository,
```

Comment:
I think that you dont need two repositories, I think you need one repository that combine both datasources. In this pr you have 2 repositories , one for each data source, but the data involved is the same, credentials and accesstokens are same data, just that from diferent layers, I think you should have only one repository with 2 data sources as depencies.

Reply:
How can it be differentiated from where it gets the credentials? It is not the same that the credentials are in the application or in the server.
Can I use two different functions in the same repository?